snippets/remote_execution_ssh_keys.erb

```diff
@@ -32,4 +32,7 @@ EOF
 chmod 700 <%= ssh_path %>
 chmod 600 <%= ssh_path %>/authorized_keys
 chown -R <%= "#{ssh_user}:" %> <%= ssh_path %>
+
+# Restore SELinux context if a linux system with restorecon:
+command -v restorecon && restorecon -RvF <%= ssh_path %>
```

Comment:
Yeah, I didn't try it in an actual provisioning, but still wondering, if the whole postinst script wouldn't break then:
```
root@ice:~# command -v restorecon && restorecon -RvF /tmp
root@ice:~# echo $?
1
```